ez_setup.py

```diff
@@ -315,13 +315,7 @@ def _extractall(self, path=".", members=None):
         self.extract(tarinfo, path)
 
     # Reverse sort directories.
-    if sys.version_info < (2, 4):
```

Comment:
ez_setup.py is a bundled script from elsewhere. I'd prefer to leave it as is.

Reply:
Thanks, I've reset the file!